Tests/DatadogTests/Datadog/Tracing/Propagation/OpenTelemetryHTTPHeadersWriterTests.swift

```diff
@@ -0,0 +1,141 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import XCTest
+@testable import Datadog
+
+class OpenTelemetryHTTPHeadersWriterTests: XCTestCase {
+    // swiftlint:disable implicitly_unwrapped_optional
+    var openTelemetryHTTPHeadersWriter: OpenTelemetryHTTPHeadersWriter!
+    var sampler: Sampler!
+    var context: OTSpanContext!
+    // swiftlint:enable implicitly_unwrapped_optional
+
+    override func setUp() {
+        super.setUp()
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
+
```

Comment:
Same, as `setUp()` and `tearDown()` are empty, it doesn't seem we need internal state.
```suggestion
```